README.md

```diff
@@ -6,58 +6,55 @@
 
 Fylla, the Norse word for `complete`, is an autocompletion script generator for the [Thor](whatisthor.com) framework.
 
-It currently generates zsh completion scripts, but will soon have support for bash scripts as well. 
+Fylla currently generates zsh completion scripts. Support for bash scripts will be added.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+Add the following line to your application's Gemfile:
 
 ```ruby
 gem 'fylla'
 ```
 
-And then execute:
+execute using the following command:
 
     $ bundle
 
-Or install it yourself as:
+Or install fylla manually:
 
     $ gem install fylla
 
 ## Usage
 
-`Fylla` must be loaded before `Thor.start` is called in order for it to be used. 
+`Fylla` must be loaded before `Thor.start` can be used. 
```

Comment:
This is changing the meaning of the sentence. Let me try to reword it. 

`Fylla must be loaded before Thor.start (the method) is called, in order for Fylla to be callable, else no completions will be generated`. Hopefully that gets the point of the sentence across (don't use that sentence, just trying to explain what I was trying to say there.